beater/cloudbeat.go

```diff
@@ -153,9 +155,35 @@ func (bt *cloudbeat) Run(b *beat.Beat) error {
 
 		case update := <-bt.configUpdates:
 			if err := bt.config.Update(update); err != nil {
-				logp.L().Errorf("could not update cloudbeat config: %v", err)
+				logp.L().Errorf("Could not update cloudbeat config: %v", err)
+			}
+
+			policies, err := csppolicies.CISKubernetes()
+			if err != nil {
+				logp.L().Errorf("Could not load CIS Kubernetes policies: %v", err)
```

Comment:
this should throw an error, no? is there any reason we would want to continue with the rest of the function execution?

Reply:
Nice catch, true for all the errors here too. Fixed.